templates/default/customer/customeradd.html

```diff
@@ -213,22 +229,123 @@ <H1>{$layout.pagetitle}</H1>
 						</div>
 					</TD>
 				</TR>
-				<TR>
-					<TD>
-						{icon name="legal-personality"}
-					</TD>
-					<TD>
-						<select name="customeradd[flags][]" {tip text="Select customer flags (optional)" class="lms-ui-multiselect"}
-								multiple>
-							{foreach Localisation::arraySort($_CUSTOMERFLAGS, 'label') as $cflag => $flag}
-								<option value="{$cflag}"
-										{if is_array($customeradd.flags) && in_array($cflag, $customeradd.flags)} selected{/if}
-										title="{$flag.tip}"
-								>{$flag.label}</option>
-							{/foreach}
-						</select>
-					</TD>
-				</TR>
+				<tr>
+					<td>{trans("SSN")}</td>
+					<td>
+						<input type="text" size="30" value="{$customeradd.ssn}" name="customeradd[ssn]"
+						    {tip trigger="ssn"}>
+						{if $customeradd.ssnwarning}
+							<input type="hidden" name="customeradd[ssnwarning]" value="1">
+						{/if}
+						{if $customeradd.ssnexistencewarning}
+							<input type="hidden" name="customeradd[ssnexistencewarning]" value="1">
+						{/if}
+					</td>
+				</tr>
+				<tr>
+					<td>{trans("TEN")}</td>
+					<td>
+						<input type="text" size="30" value="{$customeradd.ten}" name="customeradd[ten]"
+							{tip trigger="ten"}>
+						{if $customeradd.tenwarning}
+							<input type="hidden" name="customeradd[tenwarning]" value="1">
+						{/if}
+						{if $customeradd.tenexistencewarning}
+							<input type="hidden" name="customeradd[tenexistencewarning]" value="1">
+						{/if}
+						{button type="link" icon="division" class="gusapi-button"
+							data_type=Utils::GUS_REGON_API_SEARCH_TYPE_TEN
+							tip="Click here to autocomplete company data from GUS REGON database"}
+					</td>
+				</tr>
+				<tr>
+					<td>{trans("REGON")}</td>
+					<td>
+						<input type="text" size="30" value="{$customeradd.regon}" name="customeradd[regon]"
+							{tip trigger="regon"}>
+						{button type="link" icon="division" class="gusapi-button"
+							data_type=Utils::GUS_REGON_API_SEARCH_TYPE_REGON
+							tip="Click here to autocomplete company data from GUS REGON database"}
+					</td>
+				</tr>
+                <tr>
```

Comment:
Coś z wcięciami tu jest nie tak.

Reply:
Moje oczy niewyłapały tego robiąc review przed wysłaniem PR.
taby vs spacje.... https://github.com/chilek/lms/pull/2253/commits/fdd9f09639be726bc31bf68e5e245c50b099ba77

Może zrobimy z tym porządek (pozbędziemy się wcięć tabowych i białych spacji) w całym projekcie - w sam raz zmiana na 28.x? Mam już gotowe nawet na to kilkulinijkowce.

Reply:
> pozbędziemy się wcięć tabowych i białych spacji

Dla HTML nie ma chyba specjalnie narzuconego formatowania kodu pod względem wcięć, więc ciężko tak szybko zdecydować się na jedno lub drugie.

Reply:
> W PHPStorm widać gołym okiem rozróżnienie spacji od tabulatorów.

To ma działać jak automat - jak jest mix formatowań w pliku powoduje, że phpstorm czasami źle się domyśla czy mamy taby czy spacje.

> Dla HTML nie ma chyba specjalnie narzuconego formatowania kodu pod względem wcięć, więc ciężko tak szybko zdecydować się na jedno lub drugie. 

Używanie standardów jest również opcjonalne. Sugeruję zrobić jak duzi:
np. https://developers.google.com/style/html-formatting

"Don't use tabs to indent text; use spaces only. Different text editors interpret tabs differently, and some Markdown features expect spaces and not tabs.".

Co o tym myślisz?

Reply:
Skoro google tak sugeruje to znaczy, że tak trzeba będzie zrobić w pewnej godzinie 0.

Reply:
Jeszcze jedno: a może w ramach głupiego bojkotu wszystkiego, co rosyjskie i laznia dupy dla ukraińców zrezygnować też z używania phpstorm? :)

Reply:
> Skoro google tak sugeruje to znaczy, że tak trzeba będzie zrobić w pewnej godzinie 0.

A to commit przed tagiem 28.0 może być do tego dobrym momentem - nie? 
Co jest blokerem dla wypuszczeniem 28.x?

> Jeszcze jedno: a może w ramach głupiego bojkotu wszystkiego, co rosyjskie i bezmyślnego lizania dupy dla ukraińców zrezygnować też z używania phpstorm? :)

A to JetBrain nie jest z Czech?

Reply:
> A to JetBrain nie jest z Czech?

Poczytaj więcej o tej firmie, Gdzie mają placówki. Kto był założycielem, itd.

Reply:
W ramach bojkotu mam odinstalować darmowego 360-dniowego triala? :D

Reply:
Daj tylko znać czy przygotować PR (ponownie) usuwający białe spacje na końcu linii i zamienić taby na spacje we wcięciach lub sobie darować to.